packages/bundle/esbuild.static.mjs

```diff
@@ -0,0 +1,266 @@
+/// <reference types="node" />
+
+/* eslint-env node */
+/* eslint-disable no-console */
+/* eslint-disable no-magic-numbers */
+/* eslint-disable no-restricted-globals */
+
+import { build, context } from 'esbuild';
+import { resolve as importMetaResolve } from 'import-meta-resolve';
+import { dirname, resolve } from 'path';
+import { readPackageUp } from 'read-pkg-up';
+import { fileURLToPath, pathToFileURL } from 'url';
+
+// const isomorphicReactPlugin = {
```

Comment:
Can we keep it but not commented, just eslint-ignore the unused variable.

This way we can keep the code and we have to adjust it rather have it dead commented.